### PR TITLE
Checkoutpage delivery address

### PR DESCRIPTION
**Pull request by iammidhu**
Hi,

Please find the pull request for the branch checkoutpage_delivery_address  which has the following changes.
- Added new content to success page
- Included delivery address feature.

### CODE REVIEW

File: app/controllers/checkoutController.js [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-        }
-    }
-    $scope.change = function(index, changeCount, $event) {
-        $scope.info.total -= $scope.info.review[index].total;
-        $scope.info.review[index].total = changeCount * $scope.info.review[index].pricePerItem;
-        $scope.info.total += $scope.info.review[index].total;
-    }
-    $scope.success = function(){
-      $location.path('/success');
+myApp.controller('checkoutController', ['$scope', '$location', 'checkoutData', 'finalData', function($scope, $location, checkoutData, finalData) {
```
Comment (nishintn) [inline app/controllers/checkoutController.js]: Rename myApp

File: app/controllers/checkoutController.js [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+  $scope.info = data;
+  $scope.isDisabled = false;
+  $scope.deleteRow = function(index) {
+    $scope.info.total -= $scope.info.review[index].total;
+    $scope.info.review.splice(index, 1);
+    if ($scope.info.total == 0) {
+      $scope.isDisabled = true;
     }
+  }
+  $scope.change = function(index, changeCount, $event) {
```
Comment (nishintn) [inline app/controllers/checkoutController.js]: Rename function name.

File: app/controllers/successController.js [hunk 3]
```diff
@@ -1,3 +1,19 @@
-myApp.controller('successController', function($scope, $location){
+myApp.controller('successController', ['$scope', '$location', 'finalData', function($scope, $location, finalData) {
 
-});
+  var next = finalData;
+  var orderData = next.get();
+
+  $scope.total = orderData.data.total;
+  $scope.payment_types = ['pay now', 'pay on delivery'];
```
Comment (nishintn) [inline app/controllers/successController.js]: Cash on Delivery

**Comment by iammidhu** [comment]
@nishintn : Changes done according to review comments.

**Comment by iammidhu** [comment]
@nishintn : synced with master

**Pull request merged**